### PR TITLE
One-click subclass selection when nothing is selected

### PR DESCRIPTION
**Pull request by bhollis**
This somewhat supercedes #10044 - the goal here is to make it one click (and more enticing) to choose a subclass. My worry is that it might be confusing to display the subclasses like this, though - that's why I made them a bit smaller and dimmed them.

Before:
Screenshot 2023-11-05 at 5 28 53 PM

After:
Screenshot 2023-11-05 at 5 18 27 PM

### CODE REVIEW

**Comment by lowPolySkeleton** [comment]
Love this too!

File: src/app/loadout/loadout-edit/LoadoutEditSubclass.tsx [hunk 1]
```diff
@@ -22,18 +23,29 @@ import { useEquipDropTargets } from './useEquipDropTargets';
...
   const stores = useSelector(storesSelector);
+  const allItems = useSelector(allItemsSelector);
+
+  const subclassItemFilter = (item: DimItem) =>
+    item.bucket.hash === BucketHashes.Subclass &&
+    item.classType === classType &&
+    item.owner === storeId &&
+    itemCanBeInLoadout(item);
+
+  const subclassItems = allItems.filter(subclassItemFilter);
```
Comment (robojumper) [inline src/app/loadout/loadout-edit/LoadoutEditSubclass.tsx:48]: Can we sort the subclasses in a consistent way?

**Review by robojumper — APPROVED** [review]
Great!

**Review by guise — APPROVED** [review]

**Comment by arnabghosal** [comment]
This is indeed a good positive change both visually and from a UX perspective
Would add that this PR doesnt need to supercede #10044. It could be an addon to that PR.
Once a subclass is selected, it could display the icons as proposed in #10044

**Pull request merged**